README.md

```diff
@@ -0,0 +1,25 @@
+# eCommerce Bookstore
+
+## Description
+
+This is an incomplete eCommerce bookstore built with RUAH MVC PHP framework. The framework itself was developed by Curtis Parham. https://bitbucket.org/parhamcurtis/ruah-mvc-framework/src/master/ 
+
+## Prerequisites/Development Setup
+
+- To view the site on your local machine install one of the AMP solution stacks (XAMPP, LAMP, MAMP etc.) depending on your OS or install required software (Apache, MySQL/MariaDB and PHP). 
+- Create database named 'asimov' and import asimov.sql file
```

Comment:
Install XAMPP, create db, import sql and we're done?
I do only that and it will work? Of course not.

A developer who never saw your repo will need more than that.